test/python/WMComponent_t/DBS3Buffer_t/DBSBufferFile_t.py

```diff
@@ -245,24 +245,24 @@ def testLoad(self):
         testFileC.load()
 
         assert testFileA == testFileB, \
-               "ERROR: File load by LFN didn't work"
+            "ERROR: File load by LFN didn't work"
 
         assert testFileA == testFileC, \
-               "ERROR: File load by ID didn't work"
-
-        assert type(testFileB["id"]) == int or type(testFileB["id"]) == long, \
-               "ERROR: File id is not an integer type."
-        assert type(testFileB["size"]) == int or type(testFileB["size"]) == long, \
-               "ERROR: File size is not an integer type."
-        assert type(testFileB["events"]) == int or type(testFileB["events"]) == long, \
-               "ERROR: File events is not an integer type."
-
-        assert type(testFileC["id"]) == int or type(testFileC["id"]) == long, \
-               "ERROR: File id is not an integer type."
-        assert type(testFileC["size"]) == int or type(testFileC["size"]) == long, \
-               "ERROR: File size is not an integer type."
-        assert type(testFileC["events"]) == int or type(testFileC["events"]) == long, \
-               "ERROR: File events is not an integer type."
+            "ERROR: File load by ID didn't work"
+
+        assert isinstance(testFileB["id"], int) or isinstance(testFileB["id"], long), \
+            "ERROR: File id is not an integer type." )
+        assert assert isinstance(testFileB["size"], int) or isinstance(testFileB["size"], long), \
```

Comment:
Dirk, Jenkins is failing because this file has some syntax error in it, it seems, and it can't parse it. I'm guessing its the "assert assert" here.

If you are re-writing things rather than just modifying them a bit, please use self.assertTrue or self.assertEqual instead of bare asserts which are not part of the unit testing.